apps/openchallenges/apex/Caddyfile

```diff
@@ -22,6 +22,11 @@
 		reverse_proxy {env.ZIPKIN_HOST}:{env.ZIPKIN_PORT}
 	}
 
+	handle_path /mcp-server {
+    rewrite * /sse
```

Comment:
You shouldn't need this rewrite.

Reply:
this has been addressed, sorry forgot to push the commits